docker-compose.yml

```diff
@@ -0,0 +1,43 @@
+version: '3.1'
+services:
+  server:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    image: magic8bot
+    volumes:
+      - /app/
+    ports:
+      - "9999:9999"
+    expose: 
+      - 9999
+    depends_on:
+      - mongodb
+    environment:
+      - MONGODB_PORT_27017_TCP_ADDR=mongodb
+
+  mongodb:
+    image: mongo:latest
+    volumes:
+      - database:/data/db
+    command: mongod --smallfiles --bind_ip=0.0.0.0 --logpath=/dev/null
+    expose:
+      - 27017
+
+# Remove below comments to use this container. "adminMongo is a Web based user interface (GUI) to handle all your MongoDB connections/databases needs."
```

Comment:
I think, you could remove that comment?
Maybe just hint, that this is the admin connection and can be commented out with `#` if not needed.